draft-ietf-tls-esni.md

```diff
@@ -934,6 +937,17 @@ IANA is requested to create an entry, echo_required(121) in the
 existing registry for Alerts (defined in {{!RFC8446}}), with the
 "DTLS-OK" column being set to "Y".
 
+# ECHOConfig Extension Guidance {#config-extensions}
+
+Any future information or hints that influence the outer ClientHello should be
```

Comment:
SHOULD?

Reply:
I read them both the same, so that works for me.